INSTALL.md

```diff
@@ -0,0 +1,109 @@
+# Installation
+
+This document describe how to set up Orchestration/Workflow Manager for OpenSDS project.
+
+* In this document Workflow Manager used is [StackStorm](https://stackstorm.com/).
+* A Docker instance of StackStorm will be created from the StackStorms [repository](https://github.com/StackStorm/st2-docker)
+* A StackStorm Pack with OpenSDS specific workflows and actions will be loaded into it.
+* A limited set of Workflows/Actions are available in [Orchestration](https://github.com/opensds/orchestration/tree/master/contrib/st2/opensds) reposity for OpenSDS.
+* An example usage of these Workflows is listed below.
+
+#### Install OpenSDS
+
+Installation steps for OpenSDS is listed in the WIKI page of [OpenSDS](https://github.com/opensds/opensds/wiki).
+The recommended OpenSDS Local Cluster Installation may be followed for initial testing of Orchestration.
+
+
+#### Install Orchestration 
+* Install StackStorm
+	The stackstorm [docker installer](https://github.com/StackStorm/st2-docker) repo is cloned, build as below.
+    ```sh
+    $ git clone https://github.com/stackstorm/st2-docker
+    $ cd st2-docker
+    $ make env
+    ```
+* Start docker container using docker-compose
+    ```sh
+    $ docker-compose up -d  # For starting container
+    $ docker-compose down   # For stopping container
+    ```
+* Copy [opensds folder](https://github.com/opensds/orchestration/tree/master/contrib/st2/opensds) to packs folder of stackstorm docker instance.
+	```sh
+	$ docker-compose exec stackstorm /bin/bash
+	# scp -r <host opensds path> /opt/stackstorm/packs/
```

Comment:
Should we use the full path of commands like scp or st2ctl? Like /usr/bin/scp or /usr/bin/st2ctl.